include/picongpu/algorithms/Gamma.hpp

```diff
@@ -40,7 +40,7 @@ namespace picongpu
     {
         using namespace pmacc;
 
-        valueType const fMom2 = math::abs2( precisionCast< valueType >( mom ) );
+        valueType const fMom2 = pmacc::math::abs2( precisionCast< valueType >( mom ) );
```

Comment:
Do we need `pmacc::` here? I thought, basically all of PIConGPU relies on having `using namespace pmacc;` everywhere.

Reply:
The namespace `math` in PIConGPU is pointing to `cupla::math`, therefore it is required.

Reply:
I hope that we can remove `pmacc::math` soon, at the moment it is a collection of stuff which is missing in alpaka/cupla

Reply:
Ah, thanks for explaining. I assumed it points to `pmacc::math` , similarly to `pmacc::algorithms::math` before.